docs/4.4. Behaviour - Resource Servers.md

```diff
@@ -13,17 +13,22 @@ back-off mechanism to avoid overloading the Authorization Server in the event of
 Resource Server is unable to contact an Authorization Server, the Resource Server MAY assume currently held public
 keys remain valid until it is able to re-establish a connection to an Authorization Server.
 
-Resource Servers SHOULD attempt to verify tokens against every public key presented at its Authorization Server's
-`/jwks` endpoint, until the Resource Server finds a public key that verifies the token, or until no keys are left.
+Resource Servers SHOULD attempt to verify tokens against every public key presented at the Authorization Server's
```

Comment:
I wouldn't say 'every' here, as that could include keys which aren't listed under the nmos pattern, and there could be many tens or hundreds.